core/src/components/divider/divider.tsx

```diff
@@ -0,0 +1,58 @@
+import { getIonTheme } from '@global/ionic-global';
+import type { ComponentInterface } from '@stencil/core';
+import { Component, Prop, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'ion-divider',
+  styleUrls: {
+    ios: 'divider.scss',
+    md: 'divider.scss',
+    ionic: 'divider.scss',
+  },
+  shadow: true,
+})
+export class Divider implements ComponentInterface {
+  /**
+   * Set to `"xxsmall"` for the smallest spacing.
+   * Set to "xsmall" for a very small spacing.
+   * Set to `"small"` for a small spacing.
+   * Set to "medium" for a medium spacing.
+   * Set to "large" for a large spacing.
+   * Set to `"xlarge"` for the largest spacing.
+   *
+   * Defaults to `"small"` for the `ionic` theme, undefined for all other themes.
```

Comment:
```suggestion
   * Defaults to `"xxsmall"`.
```

The Figma design shows default as xxsmall, is this right?